docs/source/awesome-trio-libraries.rst

```diff
@@ -0,0 +1,64 @@
+Awesome Trio Libraries
+======================
+
+.. List of Trio Libraries
+
+   A list of libraries that support Trio, similar to the awesome-python
+   list here: https://github.com/vinta/awesome-python/
+
+
+.. currentmodule:: trio
+
+You have completed the turorial, and are enthusiastic about building
+great new applications and libraries with async functionality.
+However, to get much useful work done you will want to use some of
+the great libraries that support Trio-flavoured concurrency. This list
+is not complete, but gives a starting point. Another great way to find
+Trio-compatible libraries is to search on PyPI for the ``Framework :: trio``
+tag -> `PyPI Search <https://pypi.org/search/?q=Framework+%3A%3A+trio>`__
+
+
+Web and HTML
+------------
+* `asks <https://github.com/theelous3/asks>`__ - asks is an async requests-like http library
+* `trio-websocket <https://github.com/HyperionGray/trio-websocket>`__ - This library implements the WebSocket protocol, striving for safety, correctness, and ergonomics.
+* `quart-trio <https://gitlab.com/pgjones/quart-trio/>`__ - Like Flask, but for Trio. A simple and powerful framework for building async web applications and REST APIs. Tip: this is an ASGI-based framework, so you'll also need an HTTP server with ASGI support.
+* `hypercorn <https://gitlab.com/pgjones/hypercorn>`__ - An HTTP server for hosting your ASGI apps. Supports HTTP/1.1, HTTP/2, HTTP/3, and Websockets. Can be run as a standalone server, or embedded in a larger Trio app. Use it with ``quart-trio``, or any other Trio-compatible ASGI framework.
+
+
+Database
+--------
+
+* `triopg <https://github.com/python-trio/triopg>`__ - PostgreSQL client for Trio based on asyncpg
+* `trio-mysql <https://github.com/python-trio/trio-mysql>`__ - Pure Python MySQL Client
+* `sqlalchemy_aio <https://github.com/RazerM/sqlalchemy_aio>`__ - Add asyncio and Trio support to SQLAlchemy core, derived from alchimia.
+* `sphinxcontrib-trio <https://github.com/python-trio/sphinxcontrib-trio>`__ - Make Sphinx better at documenting Python functions and methods
```

Comment:
`sphinxcontrib-trio` doesn't have anything to do with databases :-). 

Also probably worth mentioning that it makes it easy to document async functions.